controllers/messageController.js

```diff
@@ -0,0 +1,72 @@
+const message = require('../models/MessageModel')
+
+const controllers = {
+    // Get all Messges
+    getMessages: (req, res) => {
+       message.find((err, data) => {
+        if(err){
+            return next(err) 
+        }
+        else {
+            res.json(data);
+        }
+    })
+    },
+
+    // Get Single Message
+    getOne: (req, res) => {
+        const id = req.params.id;
+        message.findById(id, (err, data) => {
+            if(err){
+                return next(err) 
+            }
+            else {
+                res.json(data);
+            }
+        })
+    },
+
+    //CREATE Messages
+    create: (req, res) => {

```

Comment:
You could be a bit more specific with the property name, for example: `createNewMessage`, in this file it is quite clear that it is to create messages, we also have the comment, but when we import that into another file it may not be so clear.